src/averageAges.js

```diff
@@ -37,7 +39,13 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const womenAge = people
+    .filter((woman) => arguments.length < 2
```

Comment:
Instead of checking `arguments.length < 2`, consider using `!withChildren` for better readability.